src/main/java/taxi/model/Driver.java

```diff
@@ -39,6 +41,22 @@ public void setLicenseNumber(String licenseNumber) {
         this.licenseNumber = licenseNumber;
     }
 
+    public String getLogin() {
+        return login;
+    }
+
+    public void setLogin(String login) {
+        this.login = login;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
```

Comment:
I'd also recommend adding new fields to the equals and hashcode methods.

Reply:
we can easily remove that methods... but ok